dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/artifactbrowser/SendItemRequestAction.java

```diff
@@ -118,7 +118,11 @@ public Map act(Redirector redirector, SourceResolver resolver, Map objectModel,
                 )
                 .getRequestItemAuthor(context, item);
 
-        RequestItem requestItem = new RequestItem(item.getID(), Integer.parseInt(bitstreamId), requesterEmail, requesterName, message, Boolean.getBoolean(allFiles));
+	// probably wrong -- Boolean.getBoolean() returns true if and only if
+	// the system property named by the argument exists and is equal to the
+	// string "true".
+        // RequestItem requestItem = new RequestItem(item.getID(), Integer.parseInt(bitstreamId), requesterEmail, requesterName, message, Boolean.getBoolean(allFiles));
```

Comment:
Whoops, I meant to do that -- I was so caught up in the process I forgot.
Next time I will do so for sure!!
Bill

On Fri, Aug 19, 2016 at 2:51 PM, Tim Donohue <notifications@github.com>
wrote:

> In dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/artifactbrowser/
> SendItemRequestAction.java
> <https://github.com/DSpace/DSpace/pull/1497#discussion_r75541847>:
>
> > @@ -118,7 +118,11 @@ public Map act(Redirector redirector, SourceResolver resolver, Map objectModel,
> >                  )
> >                  .getRequestItemAuthor(context, item);
> >
> > -        RequestItem requestItem = new RequestItem(item.getID(), Integer.parseInt(bitstreamId), requesterEmail, requesterName, message, Boolean.getBoolean(allFiles));
> > +	// probably wrong -- Boolean.getBoolean() returns true if and only if
> > +	// the system property named by the argument exists and is equal to the
> > +	// string "true".
> > +        // RequestItem requestItem = new RequestItem(item.getID(), Integer.parseInt(bitstreamId), requesterEmail, requesterName, message, Boolean.getBoolean(allFiles));
>
> Minor note... you can strip out all these comments, as they don't need to
> be in the code itself (and they just describe why you changed the below
> line).
>
> Instead, if you want to explain this difference, you can do so in the PR
> description/comments.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/DSpace/DSpace/pull/1497/files/679c971ec33126ebcbe6871835a7ac9ce408c931#r75541847>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AJqhOeOKan00Ogqo1svsFvw3Muywucr-ks5qhgkvgaJpZM4JouHX>
> .
>



-- 
Human wheels spin round and round
While the clock keeps the pace... -- John Mellencamp
________________________________________________________________
Bill Tantzen    University of Minnesota Libraries
612-626-9949 (U of M)    612-325-1777 (cell)